<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthenticationForm</class>
 <widget class="QWidget" name="AuthenticationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: #F2F3F7;
font-family: &quot;Lucida Sans Typewriter&quot;;
}

QPushButton
{
border-radius: 6px;
background-color: #28A745;
}

QPushButton:hover
{
background-color: #218838;
color: #F2F3F7;
}

QPushButton:pressed
{
background-color: #1E7E34;
border-style: solid;
border-width: medium;
}

QLineEdit
{
border: 1px solid gray;
border-radius: 8px;
padding: 0 8px;
background-color: #FFFFFF;
}

QLineEdit:focus
{
border: 2px solid #DFC99B;
}
</string>
  </property>
  <widget class="QLabel" name="EmblemLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #636363;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Sign In&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogInPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegistrationPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #B09B9C;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-style:italic;&quot;&gt;All your personal information remains confidential. Created by Bohdan Marchak&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="StateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
